docs/learn/dealing-with-latency.md

```diff
@@ -19,35 +18,41 @@ Accuracy/consistency | server rewind, server authority
 
 ---
 
-We saw in our [latency](lagandpacketloss.md) and [tick](../reference/glossary/ticks-and-update-rates) pages how waiting for your server's response makes your game feel unresponsive.
-Latency will destroy your gameplay experience and make your game frustrating. It doesn't take much to make a multiplayer game unplayable. Add 200ms between your inputs and what you see on your screen and you'll want to throw your computer through the window.
+As mentioned in [latency](lagandpacketloss) and [tick](../reference/glossary/ticks-and-update-rates), waiting for your server's response makes your game feel unresponsive.
```

Comment:
I don't know that this is true or not, but I know when I was looking at the Golden Path, removal of the *.md broke the link. Is that going to happen here?

Reply:
I played around with Docusarus links quite a bit yesterday, and it looks like you can use the file ID or the file path. It seems like using the file ID is more consistent and I was able to fix a handful of broken links by removing the `.md` extension. I'll take another look here and make sure all the links are working